trpc/router/user.ts

```diff
@@ -141,4 +141,8 @@ export const user = router({
 
         return prisma.settings.update({ where: { id: activity?.settingsId }, data: input });
     }),
+    searchUser: protectedProcedure.input(z.string()).mutation(async ({ input }) => {
+        const activity = await prisma.user.findFirst({ where: { nickname: input } });
+        return activity ? activity.nickname : null;
```

Comment:
```
return prisma.user.findFirst({ where: { nickname: input } });
```

it'll be null in case of user doesn't exists. it means on client success validation must be when result is `null`.